lib/game.rb

```diff
@@ -60,21 +60,33 @@ def convert_input(input)
   def take_turn(col)
     # Player turn
     row = @new_board.open_row[col]
-    
-    @new_board.grid["row_#{row}".to_sym][col] = "X"
+    @new_board.grid[row_sym(row)][col] = "X"
+
+    if @new_board.check_win?("X",col)
+      puts "VICTORY!!!"
+    end
+
     @new_board.update_open(col)
 
     # Computer turn
     comp_col = @new_board.open_col.sample
     comp_row = @new_board.open_row[comp_col]
+    @new_board.grid[row_sym(comp_row)][comp_col] = "O"
+
+    if @new_board.check_win?("O",comp_col)
+      puts "LOSERDOM!!!"
```

Comment:
I like the cheeky win loss messages. I think the puts statements should go in our runner file though